app/assets/stylesheets/homepage.scss

```diff
@@ -15,3 +15,9 @@
 .row {
   float: right;
 }
+
+.page-header{
+	h1{
+		text-align: center;
+	}
+}
```

Comment:
fine on my side but set tab spaced to line spaces, so should solve the problem on next pushup